tchannel-crossdock/docker-compose.yml

```diff
@@ -0,0 +1,37 @@
+version: '2'
+
+services:
+    crossdock:
+        image: crossdock/crossdock
+        links:
+            - go
+            - java
+            - python
+        environment:
+            - WAIT_FOR=go,java,python
+
+            - AXIS_CLIENT=go
```

Comment:
Makes sense, we have a couple tests like this in YARPC, where the "driver" of the test is only implemented once :)